registrations/management/commands/verify_registration_schedule.py

```diff
@@ -3,16 +3,11 @@
 
 from uuid import UUID
 from django.core.management.base import BaseCommand, CommandError
-from django.core.validators import URLValidator
 from seed_services_client import StageBasedMessagingApiClient
 
 from registrations.models import Registration
 from hellomama_registration import utils
-
-
-def validate_and_return_url(url):
-    URLValidator()(url)
-    return url
+from ._utils import validate_and_return_url
```

Comment:
underscored because it is in the management commands folder but [not a management command](https://docs.djangoproject.com/en/1.10/howto/custom-management-commands/#module-django.core.management)